Library/Formula/redir.rb

```diff
@@ -0,0 +1,15 @@
+class Redir < Formula
+  homepage "http://sammy.net/~sammy/hacks/"
+  url "https://github.com/TracyWebTech/redir/archive/2.2.1-9.tar.gz"
+  version "2.2.1_9"
+  sha1 "84ae75104d79432bbc15f67e4dc2980e0912b2b6"
+
+  depends_on "cmake" => :build
```

Comment:
Actually it's not needed. I've missed it. I'll send an updated in a few minutes.